src/features/marketplace/process/commands/history-item/store-history-item-command.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
```

Comment:
@mmiranda-stackit I had to repeat this command in minting and marketplace due to state, I think we can create a new base state and extend it